pkg/grpc/grpc.go

```diff
@@ -87,6 +88,10 @@ func NewGRPCServersFromConfigurationAndServe(configurations []*configuration.GRP
 		grpc_prometheus.EnableHandlingTimeHistogram(grpc_prometheus.WithHistogramBuckets(prometheus.DefBuckets))
 		grpc_prometheus.Register(s)
 
+		if len(configuration.ListenAddresses)+len(configuration.ListenPaths) == 0 {
+			return errors.New("GRPC server configured without any listen addresses or paths")
```

Comment:
Inside the Buildbarn code base we use gRPC error codes, even for non-gRPC service code. The reason for this is that over time, the boundary of what is service code and what is not may shift around.

Instead of using `errors.New()`, could you please use `status.Error(codes.InvalidArgument, ...)`? Thanks!